test/support/assertions.py

```diff
@@ -0,0 +1,22 @@
+import json
+from os.path import join, dirname, abspath
+from jsonschema import validate, RefResolver
```

Comment:
We should have `jsonschema` explicitly listed as our dev dependency.

Reply:
[e29584b1806da9f397ab618e8f7d2ddce7754f5e](https://github.com/Shard-Labs/starknet-devnet/pull/138/commits/e29584b1806da9f397ab618e8f7d2ddce7754f5e)

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/138/commits/2c6f53f053bb33749b29b2948fc180c45488bf7d